### DISCUSSION TITLE
Turborepo v2 output prints nothing when there are no errors with `"outputLogs": "errors-only"`

### DISCUSSION THREAD

**Discussion by mctrafik** [vercel/turborepo#8540] [Ideas]
### Verify canary release

- [X] I verified that the issue exists in the latest Turborepo canary release.

### Link to code that reproduces this issue

I'm sorry, this is too much right now.

### What package manager are you using / does the bug impact?

npm

### What operating system are you using?

Mac

### Which canary version will you have in your reproduction?

2.0.4-canary.4 

### Describe the Bug

When I run `turbo <command>` i.e. `typecheck`, it runs it in all of the package (~100 of them).

But the box output is blank, and when it disappears, all I see is this:

```
...snip...
┌ @some-company/web-platform#typecheck > 
│ 
└────>
┌ @some-company/web-reporting#typecheck > 
│ 
└────>
┌ @some-company/web-search-cli#typecheck > 
│ 
└────>
┌ @some-company/web-server#typecheck > 
│ 
└────>
┌ @some-company/web-vitals#typecheck > 
│ 
└────>
```

and this is because I have this config:

```
    "typecheck": {
      "inputs": ["src/**/*.{ts,js,tsx}", "tsconfig.json"],
      "outputLogs": "errors-only"
    },
```

### Expected Behavior

What I would expect is to see some sort of message like `no-output` instead of multiple arrows that lead nowhere in the final output.

While the commands are running, when I use arrows to be in a particular task, it's the same expectation. I should see SOMETHING indicating that the logs are suppressed.

### To Reproduce

configure any command to have `"outputLogs": "errors-only"` and run that command through turbo.

**Comment by anthonyshew** [comment]
Good idea! I'll bump this to a discussion.

**Comment by anthonyshew** [comment]
Agree, that's not as beautiful of an output as I'd love to see. 👍 

Some options to think about, along with my initial thoughts: 
- Would no output box at all for the task be preferable? I'm thinking not since I'd like to still know that the task ran.
- Writing a message for no errors into the box
```
┌ @some-company/web-vitals#typecheck > 
 │  No errors reported
└────>
```
- Reducing it down to one line (I believe this is what you're suggesting?)
```
@some-company/web-vitals#typecheck > No errors reported
```
- Something else? 🤔
